### PR TITLE
update the nightly timestamp at build time

### PR DESCRIPTION
**Pull request by jakubgs**
Resolves: https://github.com/status-im/status.im/issues/157

### CODE REVIEW

File: source/_data/nightlies.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-APK: 'https://status-im.ams3.digitaloceanspaces.com/StatusIm-181121-025924-605f8d-nightly.apk'
-IOS: 'https://i.diawi.com/1CMWB2'
-APP: 'https://status-im.ams3.digitaloceanspaces.com/StatusIm-181121-025924-605f8d-nightly.AppImage'
-MAC: 'https://status-im.ams3.digitaloceanspaces.com/StatusIm-181121-025924-605f8d-nightly.dmg'
-WIN: 'https://status-im.ams3.digitaloceanspaces.com/StatusIm-181121-025924-605f8d-nightly.exe'
+APK: 'https://status-im.ams3.digitaloceanspaces.com/StatusIm-181122-025918-a5c5e1-nightly.apk'
```
Comment (chadyj) [inline source/_data/nightlies.yml]: Is this file meant to be included?

Comment (jakubgs) [reply source/_data/nightlies.yml]: The build process updates it, there's not harm in commiting the updated links.

**Comment by chadyj** [comment]
@andytudhope what's the process to deploy this to master and update the site? It's a little unclear in the readme.
After merging to develop this command should be run `git merge develop --ff`? Who should do that?

**Comment by andytudhope** [comment]
Anyone can - it just needs to be a signed commit. Will do it now.

**Pull request merged**